Source/WebCore/dom/Document.cpp

```diff
@@ -9326,6 +9326,15 @@ std::optional<Vector<uint8_t>> Document::wrapCryptoKey(const Vector<uint8_t>& ke
     return page->cryptoClient().wrapCryptoKey(key);
 }
 
+std::optional<Vector<uint8_t>> Document::serializeAndWrapCryptoKey(CryptoKeyData&& keyData)
```

Comment:
This looks like a new API. Who is calling it ?

Reply:
`auto wrappedKey = serializeAndWrapCryptoKey(m_lexicalGlobalObject, key->data());` in Source/WebCore/bindings/js/SerializedScriptValue.cpp